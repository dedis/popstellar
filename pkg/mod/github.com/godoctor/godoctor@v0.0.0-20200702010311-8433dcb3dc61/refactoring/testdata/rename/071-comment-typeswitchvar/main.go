package main

import "fmt"

// hello

func main() {
	// hello
	var x interface{} = "hello"
	// hello
	switch hello := x.(type) { // <<<<<rename,11,9,11,9,xxx,pass
	// hello
	case int:
		// hello
		fmt.Println(hello)
		// hello
	case string:
		// hello
		fmt.Println(hello)
		// hello
	default:
		// hello
		fmt.Println(hello)
		// hello
	}
	// hello
}
