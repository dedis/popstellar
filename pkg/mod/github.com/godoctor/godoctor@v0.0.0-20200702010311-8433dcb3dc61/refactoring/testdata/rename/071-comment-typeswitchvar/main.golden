package main

import "fmt"

// hello

func main() {
	// hello
	var x interface{} = "hello"
	// hello
	switch xxx := x.(type) { // <<<<<rename,11,9,11,9,xxx,pass
	// xxx
	case int:
		// xxx
		fmt.Println(xxx)
		// xxx
	case string:
		// xxx
		fmt.Println(xxx)
		// xxx
	default:
		// xxx
		fmt.Println(xxx)
		// xxx
	}
	// hello
}
